packages/styles/scss/components/_statcard.scss

```diff
@@ -17,6 +17,10 @@
 
       @include cornercut(72px, 40px);
 
+      .right-to-left & {
```

Comment:
@Shashika6, take a look at #524 - we should be switching to using the dir attribute instead of using the .right-to-left class.

So let's use this as our first trial component, can you include this fix and also convert this component to use the dir attribute?

Reply:
Will do @bashlk 🙌

Reply:
Going forward remove any css styles that are not needed.